src/components/auth/CreateAccount.js

```diff
@@ -1,118 +1,145 @@
-import React, { Component } from "react";
```

Comment:
**_THIS IS THE BIG BOY/GIRL COMPONENT_** with like 90% of my changes/time. Take a close look!